openedx/core/djangoapps/embargo/tests/test_middleware.py

```diff
@@ -146,26 +269,3 @@ def test_always_allow_access_to_embargo_messages(self, access_point, msg_key):
             REMOTE_ADDR="192.168.10.20"
         )
         assert response.status_code == 200
-
-    @patch.dict(settings.FEATURES, {'EMBARGO': True})
-    def test_whitelist_ip_skips_country_access_checks(self):
```

Comment:
This test became redundant, after some of my test refactoring above, so I dropped it.